warehouse/ingest-scripts/src/main/resources/bin/ingest/findJars.sh

```diff
@@ -34,7 +34,7 @@ DATAWAVE_INGEST_NYCTLC_JAR=$(findJar datawave-ingest-nyctlc)
 DATAWAVE_INGEST_CORE_JAR=$(findJar datawave-ingest-core)
 DATAWAVE_INGEST_CONFIG_JAR=$(findJar datawave-ingest-configuration)
 DATAWAVE_COMMON_JAR=$(findJar datawave-common)
-DATAWAVE_BALANCERS_JAR=$(findJar datawave-balancers)
+DATAWAVE_BALANCERS_JAR=$(findJar datawave-accumulo-extensions)
```

Comment:
should we keep the name as DATAWAVE_BALANCERS_JAR, or set it to DATAWAVE_ACCUMULO_EXTENSIONS?

Reply:
I agree with changing.

Reply:
Done